Packs/CortexAttackSurfaceManagement/IncidentFields/incidentfield-ASM_-_Private_IP.json

```
@@ -15,7 +15,8 @@
             "script": "",
             "selectValues": null,
             "type": "shortText",
-            "width": 150
+            "width": 150,
+            "_comment": "The IP address of the relevant asset."
```

```suggestion
            "_comment": "Mandatory: The IP address of the relevant asset."
```
to make the same as others